src/app/files/edit-location.component.ts

```diff
@@ -6,27 +6,29 @@ import { Location } from './location';
 @Component({
     selector: 'edit-location',
     template: `
-        <div>
-            <fieldset class="name">
-                <label>Alias</label>
-                <input [(ngModel)]="model.alias" class="form-control" type="text" autofocus>
-            </fieldset>
-            <fieldset class="path">
-                <label>Physical Path</label>
-                <input [(ngModel)]="model.path" class="form-control" type="text">
-            </fieldset>
-            <fieldset>
-                <label>Permissions</label>
-                <div class="flags">
-                    <checkbox2 [(model)]="_read">Read</checkbox2>
-                    <checkbox2 [(model)]="_write">Write</checkbox2>
-                </div>
-            </fieldset>
-            <p class="pull-right">
-                <button class="ok" (click)="onOk()">{{model.id ? 'OK' : 'Create'}}</button>
-                <button class="cancel" (click)="cancel.next()">Cancel</button>
-            </p>
+<div>
+    <h4>New IIS File System Mapping</h4>
+    <p>File system mappings make directories accessible by IIS</p>
```

Comment:
This File system is not used directly by IIS. It is used only by IIS.Administration File Sytem API.
So, please adjust this text which is fit to the existing document. https://blogs.iis.net/adminapi/file-api-enhancements.